plugins/rollbar_dashboards/app/assets/stylesheets/rollbar_dashboards/application.css

```diff
@@ -15,10 +15,6 @@
   text-overflow: ellipsis;
   white-space: nowrap;
   overflow: hidden;
-  width:75%;
-  max-width:75%;
-}
-
-.rollbar-table__env {
-  width: 15%;
+  width:90%;
```

Comment:
css linter? normally space between the colon and number

Reply:
fixed ... linter should be it's own PR ... we don't have much css anyway, so not really necessary